manager_D3S.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+import kromek
+import numpy as np
+
+import time
+
+from auxiliaries import set_verbosity
+
+class Manager_D3S(object):
+    """
+    Master object for D3S device operation. 
+    
+    Prints out spectra for every interval, stores each spectra, and sums the spectra together. 
+    
+    Interval is in seconds with the default being 30 seconds.
+    """
+    
+    def __init__(self,
+                 interval=None,
+                 count=0,
+                 transport='any', 
```

Comment:
Actually you can let transport be `'usb'` by default. The other option is Bluetooth which we don't expect to use normally.